data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -4,6 +4,8 @@
 import pandas as pd
 from data_pipeline.config import settings
 
+import score.field_names as field_names
```

Comment:
nit: why not just from score import field_names?

Reply:
updated to match the import style elsewhere